src/components/app/data/hooks/useBFF.test.jsx

```diff
@@ -234,11 +185,16 @@ describe('useBFF', () => {
         },
         fallbackQueryConfig: mockFallbackQueryConfig,
       }),
-      { wrapper: Wrapper },
+      {
+        wrapper: ({ children }) => Wrapper({
+          routes: isMatchedRoute ? '/test-enterprise' : '/test-enterprise/search',
+          children,
+        }),
```

Comment:
Rendering a React component by calling it like a function is a bit atypical; functional components generally should be rendered using JSX, i.e. `<Wrapper {...props />}`.

Combined with a suggestion above regarding the `Route`, perhaps this would be written as:

```jsx
{
  wrapper: ({ children }) => (
    <Wrapper initialEntries={['/test-enterprise']}>
      {children}
    </Wrapper>
  ),
}
```